pkg/platform/authentication/auth.go

```diff
@@ -241,7 +241,7 @@ func (s *Auth) Client() *mono_client.Mono {
 	if !s.Authenticated() {
 		if fail := s.Authenticate(); fail != nil {
 			logging.Error("Trying to get the Client while not authenticated")
-			print.Error(locale.T("err_api_not_authenticated"))
+			fmt.Fprint(os.Stderr, locale.T("err_api_not_authenticated"))
```

Comment:
Why `fmt` here? Is there a reason not to use the `output` library here?

Reply:
I was still trying to avoid having a global for `output` when I wrote this, now that I see it again I might as well rewrite it to use the global, least then we're aware of the legacy dependency.